web/src/stores/index.ts

```diff
@@ -28,6 +30,7 @@ export const useRootStore = defineStore("root", {
       tags: null,
       converters: null,
       pcapOverIPEndpoints: null,
+      clientConfig: null,
```

Comment:
We can add the config defaults here instead of allowing `null`. This makes a bunch of null checks obsolete too and simplifies the code. It also allows the websocket update messages to work even when we didn't fetch the clientconfig REST endpoint yet.